src/io/fasta.rs

```diff
@@ -509,15 +533,18 @@ impl Record {
 
     /// Return the id of the record.
     pub fn id(&self) -> Option<&str> {
-        match self.header[1..].trim_right().splitn(2, ' ').nth(0) {
-            Some("") => None,
-            value => value,
+        match self.id.as_ref() {
+            "" => None,
```

Comment:
I like returning a string rather than an option. I've modified it in both parsers to do that.